### PR TITLE
engrampa: update to 1.24.1.

### PR DESCRIPTION
**Pull request by skmpz**
cross compilation broken.

### CODE REVIEW

File: srcpkgs/engrampa/template [hunk 1]
```diff
@@ -9,8 +9,9 @@ makedepends="file-devel gtk+3-devel json-glib-devel libSM-devel libarchive-devel
...
 depends="desktop-file-utils"
 short_desc="Archive manipulator for MATE"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="skmpz <dem.procopiou@gmail.com>"
 license="GPL-2.0-or-later"
 homepage="https://mate-desktop.org"
 distfiles="https://pub.mate-desktop.org/releases/${version%.*}/${pkgname}-${version}.tar.xz"
-checksum=d415f486edc9a5474024b3928ff1127ba4534a703a60993a3912fffcf0988a8d
+checksum=fe24e04efb34a67374d2a46a3bf2a0a445b77e0ec43216e4c3f14e5cd0b7722a
+nocross="cannot run test program while cross compiling"
```
Comment (Piraty) [inline srcpkgs/engrampa/template:17]: this is unacceptable, please file an issue upstream. commit that broke it: https://github.com/mate-desktop/engrampa/commit/25b2bdbcfb95c91a33fd62370cac28a863567c3f

**Review by Piraty — CHANGES_REQUESTED** [review]

File: srcpkgs/engrampa/template (hunk 1 above)
Comment (ericonr) [reply srcpkgs/engrampa/template:17]: If they want to keep their (imo terrible) test, they should at least run it with host CC.

Comment (ericonr) [reply srcpkgs/engrampa/template:17]: Alternatively, they can just check for the two possible ZSTD values and not require all this machinery.

**Comment by sgn** [comment]
On 2020-11-12 13:19:14-0800, Érico Nogueira Rolim <notifications@github.com> wrote:
> @ericonr commented on this pull request.
> 
> 
> 
> >  license="GPL-2.0-or-later"
>  homepage="https://mate-desktop.org"
>  distfiles="https://pub.mate-desktop.org/releases/${version%.*}/${pkgname}-${version}.tar.xz"
> -checksum=d415f486edc9a5474024b3928ff1127ba4534a703a60993a3912fffcf0988a8d
> +checksum=fe24e04efb34a67374d2a46a3bf2a0a445b77e0ec43216e4c3f14e5cd0b7722a
> +nocross="cannot run test program while cross compiling"
> 
> Alternatively, they can just check for the two possible ZSTD values and not require all this machinery.

The check is terrible, let's patch it.

-- 
Danh

**Comment by Piraty** [comment]
ping @skmpz

**Comment by skmpz** [comment]
yeah, how should we proceed with this? I saw you commented on the commit upstream, but no changes yet.

**Comment by ndowens** [comment]
@skmpz Here is a patch I did that allowed me to cross-build https://termbin.com/s44og

**Comment by sgn** [comment]
On 2020-11-25 12:25:29-0800, Nathan <notifications@github.com> wrote:
> @skmpz Here is a patch I did that allowed me to cross-build https://termbin.com/s44og

That patch is awful and incorrect.

**Comment by ndowens** [comment]
> On 2020-11-25 12:25:29-0800, Nathan ***@***.***> wrote: @skmpz Here is a patch I did that allowed me to cross-build https://termbin.com/s44og
> That patch is awful and incorrect. Because, it still compiles and run, but it'll always run into failure in cross-compiling mode and give the meaningless result.

OK to learn, how do you suggest to fix it then?

**Comment by Piraty** [comment]
> yeah, how should we proceed with this? I saw you commented on the commit upstream, but no changes or replies yet. Shall we just patch it reverting the changes of that commit ?

@skmpz  as said before, file an upstream issue

**Pull request closed without merging**